.github/workflows/copilot_deploy.yml

```diff
@@ -74,3 +84,12 @@ jobs:
     - name: Copilot deploy
       run: |
         copilot svc deploy --env ${{ inputs.environment || 'test' }}
+  post_deploy_tests:
+    needs: copilot_build
+    secrets:
+      E2E_PAT: ${{secrets.E2E_PAT}}
+      GOV_NOTIFY_API_KEY: ${{secrets.GOV_NOTIFY_API_KEY}}
+    uses: communitiesuk/funding-service-design-workflows/.github/workflows/post-deploy.yml@main
+    with:
```

Comment:
Needs adding:

```app_name: ${{ github.event.repository.name }}```

Reply:
Given it uses it like this:
https://${{inputs.app_name}}-test.access-funding.test.levellingup.gov.uk/'
Shouldn't app_name just be notification?

Reply:
yeah, it can just be notification. Probably better, actually, in case we change the name of the repository.